guide/xml/portfile-variants.xml

```diff
@@ -171,9 +171,9 @@
           different tasks depending on the version of Darwin, the core
           operating system underlying macOS.
           <replaceable>version</replaceable> is the major version of Darwin,
-          and can be <literal>17</literal> for macOS High Sierra v10.13,
-          <literal>16</literal> for macOS Sierra v10.12, <literal>15</literal>
-          for OS X El Capitan v10.11, and so on.</para>
+          and can be <literal>18</literal> for macOS Mojave v10.14,
+          <literal>17</literal> for macOS High Sierra v10.13, <literal>16</literal>
+          for macOS Sierra v10.12, and so on.</para>
```

Comment:
While we are at it, could we please also drop the `v` prefix from the version numbers here? That is very unusual.

Reply:
I don't care either way, but we may have used the `v` prefix because Apple often does so; see for example [Security Update 2017-001 macOS High Sierra v10.13](https://support.apple.com/kb/DL1943?locale=en_US).

Reply:
I'll go ahead and remove it for consistency (this is the only place `v` is used).